### ISSUE TITLE
ESPhome viewbox in the Integration page is very small

### ISSUE THREAD

**Issue by manju-rn** [home-assistant/frontend#6111]
## Checklist

- [ Y] I have updated to the latest available Home Assistant version.
- [ Y] I have cleared the cache of my browser.
- [Y ] I have tried a different browser to see if it is related to my browser.

## Environment

- Home Assistant release with the issue:
- Last working Home Assistant release (if known):
- Browser and browser version:
- Operating system:

## State of relevant entities

```yaml

```

## Problem-relevant configuration

```yaml

```

## Javascript errors shown in your browser console/inspector

```txt

```

## Additional information
1. Check the snapshot of the esphome viewbox in the integration page. It is very small and required quite a scrolling even for 5 or more devices. 
![image](https://user-images.githubusercontent.com/26291847/83932232-dd3e3100-a7d3-11ea-8de5-45023b736974.png)
2. Check the entire page - so much browser real estate is there for grabbing. There should be a way to expand that view box to fill the entire browser
![image](https://user-images.githubusercontent.com/26291847/83932267-0a8adf00-a7d4-11ea-99db-482c0f7ea070.png)

**Comment by SeanPM5** [comment]
This is not a bug, it's an intentional design decision to keep all of the cards a consistent size.

In your case you only have six integrations so they all appear in one row with lots of empty space underneath. But if you had another row of cards underneath that, you wouldn't want them to be different heights, because it would waste tons of space.

Here is a quick demonstration of how it'd look:

![image](https://i.imgur.com/7YfQipd.jpg)

**Comment by manju-rn** [comment]
Agree. However, i think there should be a way to "expand" it to take full screen

**Comment by zsarnett** [comment]
Option to expand the integration into a dialog

**Comment by flixlix** [comment]
I noticed the exact same issue, when scrolling down on ESPHOME devices, some are missing in between a scroll. Any updates on this?
